h/templates/5xx.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
+      <meta name="viewport" content="width=device-width,initial-scale=1" />
+      <title>Server Error</title>
+      <style>
```

Comment:
I'm not sure. The amount that has to go wrong for image serving to break is substantially larger than the amount that has to go wrong for CSS to break, as our CSS is generated/compiled/whatever.

I think this is the right balance.